### ISSUE TITLE
Babel not works correctly

### ISSUE THREAD

**Issue by vamp** [SteveSanderson/generator-ko#19]
`if (!babelCore.canCompile(pathname)) return callback();`
returns error:

```
stream.js:94
      throw er; // Unhandled stream error in pipe.
            ^
TypeError: undefined is not a function
    at babelTranspile (...\gulpfile.js:134
:20)
```

**Comment by MarkDeVerno** [comment]
I'm having the same issue.  There were some major changes to babel recently.

**Comment by sdluxeon** [comment]
@MarkDeVerno what was your fix?

**Comment by chrconlo** [comment]
Anybody find a solution for this? Spent several hours setting this up only to hit issue #24 (which would be good to resolve). Trying to workaround by compiling via Gulp but breaks here.

**Comment by MarkDeVerno** [comment]
@chrconlo @sdluxeon  I gave up on a fix and just rolled back to a previous version.

**Comment by chrconlo** [comment]
Yes, I went back to Nov 2014 commit and used _npm link_ from local copy and all works well now. Thanks

**Comment by paulbonner** [comment]
chrconlo is right. Rolling back to d88af3ad72d72e907c5d681e38ab6c7ebf06acd6 is the answer.

**Comment by Kevinio04** [comment]
Hey guys, there is a fix to this I found on SO. Well... there is a fix to getting the Gulp build to work anyway. Its a pretty simple fix and relates to how slashes are handled between windows and OSX. 

At the top of gulpfile.js you need to add the following variable to the gulp and plugins variables:
slash = require('slash');

Then at the bottom of the file edit the babelTranspile function. Add a new line at the top of the function and paste the following:
pathname = slash(pathname);

This will allow the build to execute and the application will load from the dist folder. I have no idea how steve is running from the src folder in the demo however. No browsers support ES6 so the imports declarations in the startup file in my mind will never work, they would have to be updated to ES5 require statements. 

Original answer to the gulp problem found here:

http://stackoverflow.com/questions/33198584/gulp-failing-default-task-due-to-jquery-error-for-generator-ko

**Comment by dipaktelangre** [comment]
@Kevinio04 I spend almost day to resolve same issue. Thanks for your solution, its works :) 👍
